os/fs/smartfs/smartfs_utils.c

```diff
@@ -1246,8 +1255,8 @@ int smartfs_finddirentry(struct smartfs_mountpt_s *fs, struct smartfs_entry_s *d
 										readwrite.count = fs->fs_llformat.availbytes;
 										readwrite.buffer = (uint8_t *)fs->fs_chunk_buffer;
 
-										ret = FS_IOCTL(fs, BIOC_READSECT, (unsigned long)&readwrite);
-										if (ret < 0) {
+										tmp = FS_IOCTL(fs, BIOC_READSECT, (unsigned long)&readwrite);
+										if (tmp < 0) {
 											fdbg("Error reading sector %d header, ret : %d\n", readwrite.logsector, ret);
```

Comment:
Can you please verify if the ret value printed above should be tmp instead of ret ?

Reply:
Updated, thanks